frontend/taipy-gui/src/components/Taipy/Menu.tsx

```diff
@@ -55,41 +55,59 @@ const Menu = (props: MenuProps) => {
                 });
             }
         },
-        [onAction, dispatch, active, module]
+        [active, dispatch, module, onAction]
     );
 
     const openHandler = useCallback((evt: MouseEvent<HTMLElement>) => {
         evt.stopPropagation();
         setOpened((o) => !o);
     }, []);
 
+    const selected = useMemo(() => {
+        const selected = props.selected ?? [];
```

Comment:
why ?? and not || ?

Reply:
or even better `const selected = Array.isArray(props.selected) ? props.selected : [];`

Reply:
updated